docs/frontend/data.md

```diff
@@ -146,3 +146,43 @@ hass.callApi('delete', 'notify.html5', { subscription: 'abcdefgh' });
 :::info
 We're moving away from API calls and are migrating everything to `hass.callWS(message)` calls.
 :::
+
+## Entity state formatting
+
+These method allows you to format the state and attributes of an entity. The value will be localized using user profile settings (language, number format, date format, timezone) and unit of measurement.
+
+### `hass.formatEntityState(stateObj, state)`
+
+Format the state of an entity. You need to pass the entity state object.
+
+```js
+hass.formatEntityState(hass.states["light.my_light"]); // "On"
+```
+
+You can force the state value using the second optional parameter.
+
+```js
+hass.formatEntityState(hass.states["light.my_light"], 'off'); // "Off"
+```
+
+### `hass.formatEntityAttributeValue(stateObj, attribute, value)`
+
+Format the attribute value of an entity. You need to pass the entity state object and the attribute name.
+
+```js
+hass.formatEntityAttributeValue(hass.states["climate.thermostat"], "current_temperature"); // "20.5 °C"
+```
+
+You can force the state value using the second optional parameter.
+
+```js
+hass.formatEntityAttributeValue(hass.states["climate.thermostat"], "current_temperature", 18); // "18 °C"
+```
+
+### `hass.formatEntityAttributeName(stateObj, attribute)`
+
+Format the attribute name of an entity. You need to pass the entity state object and the attribute name.
+
+```js
+hass.formatEntityAttributeName(hass.states["climate.thermostat", "current_temperature"], ); // "Current temperature"
```

Comment:
Is this syntax correct, ie including the attribute name inside the brackets?

Reply:
No, it is not. Thanks!